data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/SimpleDataService.java

```diff
@@ -94,7 +94,8 @@ DataReaderRequest constructReaderRequest(final ReadRequest request) {
         getConfig.setOriginalRequestId(request.getOriginalRequestId());
         LOGGER.info("Calling palisade service with: {}", getConfig);
 
-        final DataRequestConfig config = getPalisadeService().getDataRequestConfig(getConfig).join();
+        // If this throws an exception connecting to the palisade-service, then it will be caught and audited
```

Comment:
Where is exception caught and audited?

Reply:
In the calling method that catches the exception